.github/workflows/build-test.yml

```diff
@@ -31,26 +31,6 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  test-install-unix:
```

Comment:
We still want to test install here. On release we want to test install and functioning

Reply:
i created a separate workflow for installer testing, And release workflow have it's own job for testing installer with latest release